modules/conversion-center/module.php

```diff
@@ -109,24 +113,46 @@ public function __construct() {
 
 		add_action( 'admin_init', function () {
 			$action = filter_input( INPUT_GET, 'action' );
-
-			if ( 'set_as_homepage' === $action ) {
-				$post = filter_input( INPUT_GET, 'post', FILTER_VALIDATE_INT );
-				check_admin_referer( 'set_as_homepage_' . $post );
-				update_option( 'page_on_front', $post );
-				update_option( 'show_on_front', 'page' );
-				$menu_args = $this->get_links_menu_args();
-
-				wp_redirect( $menu_args['menu_slug'] );
-				exit;
+			$menu_args = $this->get_contact_menu_args();
+
+			switch ( $action ) {
+				case 'remove_from_entire_site':
+					$post = filter_input( INPUT_GET, 'post', FILTER_VALIDATE_INT );
+					check_admin_referer( 'remove_from_entire_site_' . $post );
+					delete_post_meta( $post, '_elementor_conditions' );
+					wp_redirect( $menu_args['menu_slug'] );
+					exit;
+				case 'set_as_entire_site':
+					$post = filter_input( INPUT_GET, 'post', FILTER_VALIDATE_INT );
+					check_admin_referer( 'set_as_entire_site_' . $post );
+					update_post_meta( $post, '_elementor_conditions', [ 'include/general' ] );
```

Comment:
Should you not do the delete first, and then set this afterwards? it seems like you will set it and then just delete it again 🤔

Reply:
@louiswol94 I was excluding the post but switched that